code/style.css

```diff
@@ -1,4 +1,308 @@
 html {
-  background: #0025ff;
-  color: #fff;
+  background: rgb(204, 204, 204);
+  color: rgb(0, 0, 0);
+  font-weight: normal;
+}
+
+body{
+  margin: 0px;
+  font-family: 'Raleway', sans-serif;
+}
+
+/*Header and navbar*/
+
+.flex-container {
+  height:75px;
+  background-color:darkgrey;
+  color:black;
+  font-size: 25px;
+  display:flex;
+  align-items: center;
+  justify-content: space-evenly;
+  flex:wrap;
+}
+
+.logo{
+  height:150px;
+}
+
+
+ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+li {
+  display: inline;
+}
+
+a{
+  text-decoration: none;
+  font-size: 30px;
+  font-weight: bold;
+  padding:15px;
+  color:black;
+}
+
+a:hover {
+  opacity: 0.5;
+}
+
+
+/*Big news*/
+
+.big-news {
+  display: flex;
+  flex-direction: row;
+  margin:20px;
+  background-color: rgb(204, 204, 204);
+}
+
+
+.column_1 {
+  flex: 50%;
+  padding: 10px;  
+}
+
+.column_2{
+  flex: 25%;
+  padding: 10px;
+  margin:100px 300px 0px 0px;
+}
+
+.main-photo{
+  max-width:100%;
+  border-radius: 20px; 
+  box-shadow: 10px -16px darkgray;
+}
+
+    h1{
+    font-size: 35px;
+    font-weight: bold;
+    margin:20px 5px 5px 5px;
+    }
+
+    p{
+      font-size: 25px;
+      margin:20px 5px 5px 5px;
+      font-weight: bold;
+    }
+
+    h2{
+      font-size: 40px;
+    }
+
+    h4{
+      font-size: 25px;
+      margin-left: 10px;
+      margin-bottom: 10px;
+      margin-top: 10px;
+      font-weight: normal;
+    }
+
+  h3{
+    font-size: 25px;
+    font-weight: bolder;
+    margin-left: 10px;
+    margin-right: 10px;
+    margin-top: 10px;
+  }
+
+/*Small news*/
+
+.articles{
+  display:flex;
+  flex-direction: row;
+  margin:20px;
+  padding:20px;
+  background-color: rgb(204, 204, 204);
+  justify-content: space-evenly;
+  flex-wrap: wrap;
+}
+
+.content{
+  height:500px;
+  width:400px;
+  background-color: white;
+  border-radius: 20px;
+  box-shadow: 10px -16px darkgray;
+  margin-top: 40px;
+}
+
+.article_photo{
+  height:auto;
+  width:400px;
+  border-radius: 20px;
+}
+
+/*Footer*/
+
+footer {
+  background: darkgray;
+  margin-top: 1.5em;
+  padding: 0.75em;
+  font-size: 1.5em;
+  display: flex;
+  justify-content: center;
+  
+}
+
+.fa {
+  padding: 20px;
+  font-size: 30px;
+  width: 50px;
+  text-align: center;
+  text-decoration: none;
+  margin: 5px 2px;
+}
+
+.fa:hover {
+    opacity: 0.7;
+}
+
+.fa-facebook {
```

Comment:
I will maybe borrow your styling with the background of socials icons, it's pretty with the round black background.